lib/Service/ConfigService.php

```diff
@@ -560,6 +560,8 @@ public function configureRequest(NC19Request $request) {
 //		if ($this->getAppValue(ConfigService::CIRCLES_NON_SSL_LOCAL) === '1') {
 		$request->setLocalAddressAllowed(true);
 //		}
+
+
```

Comment:
I guess this is not needed.

Reply:
was not sure at the time, but i'll remove the commented line